configure.ac

```diff
@@ -1633,6 +1637,10 @@ AS_IF([test x"$enable_spacetime" != "xyes" ],
       AS_IF([$libunwind_requested && ! $libunwind_available],
         [AC_MSG_ERROR([libunwind was requested but can not be found])])
 
+      AS_IF([$libunwind_available],
+            [mkexe="$mkexe $libunwind_link_flags"
+             mksharedlib="$mksharedlib $libunwind_link_flags"])
```

Comment:
This can't be done both here and in the compiler, as it results in them being passed twice (try `ocamlopt.opt -o foo foo.ml`) and also being passed to `ocamlc -custom`. Is it enough to put them in `$nativecclibs` instead? That should propagate through the compiler's own build system correctly - it should also mean that `asmlink.ml` doesn't have to do anything.